packages/help-center/src/components/help-center-a4a-contact-form.tsx

```diff
@@ -0,0 +1,214 @@
+/* eslint-disable no-restricted-imports */
+/**
+ * External Dependencies
+ */
+import { recordTracksEvent } from '@automattic/calypso-analytics';
+import { FormInputValidation } from '@automattic/components';
+import { Button, SelectControl, TextareaControl, TextControl } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { useState, useEffect, useMemo, useCallback } from 'react';
+import { useNavigate } from 'react-router-dom';
+/**
+ * Internal Dependencies
+ */
+import { useHelpCenterContext } from '../contexts/HelpCenterContext';
+import './help-center-a4a-contact-form.scss';
+import { useSubmitA4ATicketMutation } from '../data/use-submit-a4a-support-ticket';
+
+export const HelpCenterA4AContactForm = () => {
+	const { currentUser, agency } = useHelpCenterContext();
+	const navigate = useNavigate();
+
+	const [ formData, setFormData ] = useState( {
+		name: currentUser?.display_name ?? '',
+		email: currentUser?.email ?? '',
+		site: '',
+		product: 'a4a',
+		message: '',
+		pressable_contact: 'sales',
+	} );
+
+	const [ hasSubmitError, setHasSubmitError ] = useState( false );
```

Comment:
I think the mutation gives you the error, no need to have your own state.

Reply:
This has been fixed. Thanks for pointing this 👍